### PR TITLE
CSCQVAIN-122: Issued field

### PR DESCRIPTION
**Pull request by tahme**
Show Issued field in the Content Description tab.

Also:
- allow wrapping date fields (i.e. show a rectangle around the component if the 'wrapped' prop is set)
- allow clearing date fields (fixes CSCQVAIN-121)
- show info icon for date fields

### CODE REVIEW

File: src/schemas/fairdata-ida.ui.js [hunk 1]
```diff
@@ -55,6 +55,16 @@ export default {
 			'tabFormat': false,
 		},
 	},
+	'/properties/issued': {
+		'tab': 'description',
```
Comment (jppiiroinen) [inline src/schemas/fairdata-ida.ui.js:59]: Was this changed on purpose

File: src/schemas/fairdata-ida.ui.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default {
 		{ label: 'Extra', uri: null },
 	],
 	//'': { 'tab': 'description' },
-	'': { 'tab': 'extra', 'order': ["title", "description", "language"] },
+	'': { 'tab': 'extra', 'order': ["title", "description", "issued", "language"] },
```
Comment (jppiiroinen) [inline src/schemas/fairdata-ida.ui.js:12]: why this change, the /properties/issued seems to be marked for the `description` tab which was a change also from the previous value. see other comment.

**Review by jppiiroinen — CHANGES_REQUESTED** [review]

File: src/schemas/fairdata-ida.ui.js (hunk 2 above)
Comment (tahme) [reply src/schemas/fairdata-ida.ui.js:12]: This makes "issued" field appear after description and before language.

It's a bit misleading at first, but the tab and order are unrelated here. The line does two separate things for the dataset root:
- 'tab': 'extra' makes 'extra' the default tab. Items that don't have tab set will be in the 'extra' tab, which is hidden. 
- 'order': [...] defines the order in which the items appear. First, we show items in the array order and then the remaining items in their default order. This is shared across all tabs.

File: src/schemas/fairdata-ida.ui.js (hunk 1 above)
Comment (tahme) [reply src/schemas/fairdata-ida.ui.js:59]: Yes. It was in the hidden 'extra' tab but it turns out the it's required for PAS stuff so now we show it to the user in the 'description' tab.

File: src/schemas/fairdata-ida.ui.js (hunk 2 above)
Comment (jppiiroinen) [reply src/schemas/fairdata-ida.ui.js:12]: Thanks for explaining this part 👍

**Review by jppiiroinen — APPROVED** [review]
